src/Sidekick.Business/Parsers/ItemParser.cs

```diff
@@ -217,23 +238,82 @@ private Item GetRareItem(ItemProperties itemProperties, string[] lines)
+                item.Influence = GetInfluenceType(lines.LastOrDefault());
             }
 
             return item;
         }
 
-        private Item GetUniqueItem(ItemProperties itemProperties, string[] lines)
+        private EquippableItem GetEquippableItem(string[] lines)
         {
-            var item = new EquippableItem
+            EquippableItem item;
+
+            // Weapon
+            if(lines.Any(c => c.StartsWith(languageProvider.Language.DescriptionPhysicalDamage) || c.StartsWith(languageProvider.Language.DescriptionElementalDamage)))
             {
-                Name = lines[1],
-                Type = itemProperties.IsIdentified ? lines[2] : lines[1]
-            };
+                // TODO Make Available in Query Parameters
+                item = new WeaponItem()
+                {
+                    AttacksPerSecond = GetNumberFromString(lines.Where(c => c.StartsWith(languageProvider.Language.DescriptionAttacksPerSecond)).FirstOrDefault()),
+                    CriticalStrikeChance = GetNumberFromString(lines.Where(c => c.StartsWith(languageProvider.Language.DescriptionCriticalStrikeChance)).FirstOrDefault()),
+                    ElementalDamage = GetNumberFromString(lines.Where(c => c.StartsWith(languageProvider.Language.DescriptionElementalDamage)).FirstOrDefault(), allowRange: true),
+                    PhysicalDamage = GetNumberFromString(lines.Where(c => c.StartsWith(languageProvider.Language.DescriptionPhysicalDamage)).FirstOrDefault(), allowRange: true),
+                };
+            }
+            else        // Armour or Jewellery
+            {
+                item = new ArmourItem()
+                {
+                    Armour = GetNumberFromString(lines.Where(c => c.StartsWith(languageProvider.Language.DescriptionArmour)).FirstOrDefault()),
+                    EnergyShield = GetNumberFromString(lines.Where(c => c.StartsWith(languageProvider.Language.DescriptionEnergyShield)).FirstOrDefault()),
+                    Evasion = GetNumberFromString(lines.Where(c => c.StartsWith(languageProvider.Language.DescriptionEvasion)).FirstOrDefault())
+                };
+            }    
+
+            item.ItemLevel = GetNumberFromString(lines.Where(c => c.StartsWith(languageProvider.Language.DescriptionItemLevel)).FirstOrDefault());
+
+            // TODO Special check for 1 handed weapon or shield
+
+            if(int.Parse(item.ItemLevel) >= 50)
+            {
+                item.MaxSockets = 6;
+            }
+            else if(int.Parse(item.ItemLevel) >= 35)
+            {
+                item.MaxSockets = 5;
+            }
+            else if(int.Parse(item.ItemLevel) >= 25)
+            {
+                item.MaxSockets = 4;
+            }
+            else if(int.Parse(item.ItemLevel) >= 1)
+            {
+                item.MaxSockets = 3;
+            }
+            else
+            {
+                item.MaxSockets = 1;
+            }
```

Comment:
Parse the item level once and then use a switch expression (like above for item rarity)